videoframegrid.py

```diff
@@ -68,6 +71,28 @@ def swap_frame(self, frame):
             else:
                 self.selected_frame = frame
 
+    def toggle_fullscreen(self, selected_frame):
+        if not self.fullscreen:
```

Comment:
I am generally against inverting if-statements, but you have a reasonable argument so go for it.